sdcm/nemesis.py

```diff
@@ -167,6 +167,10 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     kubernetes: bool = False        # flag that signal that nemesis run with k8s cluster
     limited: bool = False           # flag that signal that nemesis are belong to limited set of nemesises
     has_steady_run: bool = False    # flag that signal that nemesis should be run with perf tests with steady run
+    tombstone_gc_standard_mode: bool = False  # A flag denotes a nemesis impacts tombstone and its GC,
+    # i.e toggle_table_standard_gc_mode, repair, reboot, major compaction.
+    tombstone_gc_advanced_mode: bool = False  # A flag denotes a nemesis impacts tombstone and its GC,
+    # i.e modify_table_gc_mode, repair, reboot, major compaction.
```

Comment:
both flags apparently are doing the very same (at least, their description are the same), so please, remove one of them to avoid code duplication

Reply:
They just have a single crucial difference: toggle_table_standard_gc_mode Vs. modify_table_gc_mode.

Reply:
i see that, but both nemeses seem to be the same, where `toggle` is much more limited than the other one... why do we need it?

Reply:
in this case, you are running 1 single nemesis, during the whole test, so instead of `SisyphusMonkey` you can use the class name of the nemesis you want to use for that test... it doesn't make sense to add a "label" to the nemesis, to run just it during the test, does it?

Reply:
@fgelcer , it is a list of nemeses not a single one.